### PR TITLE
Add content to API reference

### PR DESCRIPTION
**Pull request by kneal**

### CODE REVIEW

**Review by plyr4 — DISMISSED** [review]
lgtm

File: content/usage/reference/api/step/get.md [hunk 1]
```diff
@@ -3,20 +3,20 @@ title: "Get"
...
 ```
 
 | Param | Description |
 |---|---|
-
-
-`{soon}` - documentation coming soon
+| org | Name of a organtization. |
+| repo | Name of a repository. |
+| build | Number of build. |
```
Comment (wass3r) [inline content/usage/reference/api/step/get.md:20]: Missing `step` param

File: content/usage/reference/api/step/remove.md [hunk 2]
```diff
@@ -26,12 +27,11 @@ Documentation Coming Soon!
...
+| 401 | Indicates the user does not have proper permissions. |
 
 ## Example Response Body
 
-```json
-{
-  "docs": "Coming soon"
-}
+```
+step github/octokitty/1/1 removed`
```
Comment (wass3r) [inline content/usage/reference/api/step/remove.md]:
```suggestion
step github/octokitty/1/1 removed
```

File: content/usage/reference/api/service/remove.md [hunk 3]
```diff
@@ -26,12 +27,11 @@ Documentation Coming Soon!
...
+| 401 | Indicates the user does not have proper permissions. |
 
 ## Example Response Body
 
-```json
-{
-  "docs": "Coming soon"
-}
+```
+service github/octokitty/1/1 removed`
```
Comment (wass3r) [inline content/usage/reference/api/service/remove.md]:
```suggestion
service github/octokitty/1/1 removed
```

**Review by wass3r — CHANGES_REQUESTED** [review]
Nice content 👍 🎉 

I had 89 suggested minor changes and github choked on submission :(. In a nutshell:

```
:s/a organtization/an organization/g
:s/Learn how to  /Learn how to /g
:s/Number of build/Number of a build/g
:s/Name of the type/Name of a type/g
:s/Name of a engine/Name of an engine/g
:s/Name of the secret/Name of a secret/g
:s/Number of service/Number of a service/g
:s/Number of step/Number of a step/g
```

Maybe worth deciding on `the` vs `a`. I presumed `a` since it was more prelevant.

**Review by jbrockopp — DISMISSED** [review]
LGTM

File: content/usage/reference/api/step/get.md (hunk 1 above)
Comment (wass3r) [reply content/usage/reference/api/step/get.md:20]: was going to say that the path looked wrong - looks like it got fixed

**Review by wass3r — APPROVED** [review]

**Pull request merged**